RocketChat/Rocket.Chat.ReactNative
# Regression: Create private team and channel as default

## Proposed changes

When creating a team or channel the default value of the type switch is True and this should create a private team or channel, however, the default value of the type in the state is "false", creating a public channel/team.

https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4381

## Issue(s)	


## How to test or reproduce

- Go to Create a channel
- You shouldn't change the Private switch value.
- Create the channel
- The channel should be PRIVATE as default

## Screenshots

## Types of changes




## Checklist



## Further comments
